dqsegdb2/http.py

```diff
@@ -48,10 +62,57 @@ def request(url, **urlopen_kw):
     """
     if urlparse(url).scheme == 'https' and 'context' not in urlopen_kw:
         from ssl import create_default_context
-        from gwdatafind.utils import find_credential
         urlopen_kw['context'] = context = create_default_context()
-        context.load_cert_chain(*find_credential())
-    return urlopen(url, **urlopen_kw)
+        req = Request(url)
```

Comment:
Can this line be pulled outside of the `if` statement, it seems to be used in all instances?